tests/base_tests.py

```diff
@@ -15,5 +16,8 @@ def tearDown(self):
     def generate_random_text():
         return j.data.idgenerator.chars(10)
 
+    def random_name(self):
```

Comment:
Please update the branch from development and remove this method as it is already in `base_tests`